Xamarin.Forms.Platform.Android/Renderers/EntryRenderer.cs

```diff
@@ -14,7 +14,28 @@
 
 namespace Xamarin.Forms.Platform.Android
 {
-	public class EntryRenderer : ViewRenderer<Entry, FormsEditText>, ITextWatcher, TextView.IOnEditorActionListener
+	public class EntryRenderer : EntryRendererBase<FormsEditText>
+	{
+		public EntryRenderer(Context context) : base(context)
+		{
+		}
+
+		[Obsolete("This constructor is obsolete as of version 2.5. Please use EntryRenderer(Context) instead.")]
+		public EntryRenderer()
+		{
+			AutoPackage = false;
+		}
+
+		protected override FormsEditText CreateNativeControl()
+		{
+			return new FormsEditText(Context);
+		}
+
+		protected override EditText EditText => Control;
+	}
+
+	public abstract class EntryRendererBase<TControl> : ViewRenderer<Entry, TControl>, ITextWatcher, TextView.IOnEditorActionListener
```

Comment:
Move this to a separate file.

Reply:
We will lose the entire files history. Just an FYI.

Reply:
Then leave it.